build.py

```diff
@@ -69,10 +69,10 @@
 # incorrectly load the other version of the openvino libraries.
 #
 TRITON_VERSION_MAP = {
-    "2.49.0dev": (
-        "24.08dev",  # triton container
-        "24.07",  # upstream container
-        "1.18.1",  # ORT
+    "2.49.0": (
+        "24.08",  # triton container
+        "24.08",  # upstream container
+        "1.19.0",  # ORT
```

Comment:
This version is not confirmed yet.

Reply:
Ok so this version is OK for Linux. 
For windows we will have to carry a side branch with version `1.18.1`